### ISSUE TITLE
Add initial states to reducers

### ISSUE THREAD

**Issue by AakashMallik** [fossasia/chat.susi.ai#1732]
**Actual Behaviour**

We now only have initial state of app.

**Expected Behaviour**

Shift initial state from flux to redux reducers

Yes

**Comment by AakashMallik** [comment]
@akshatnitd @anshumanv  @Akshat-Jain  Sir, should we have initial states component-wise or should we have a different approach?

**Comment by anshumanv** [comment]
I think we should group as per the domain, eg auth reducer could be one, messages reducer, settings reducer etc.

**Comment by AakashMallik** [comment]
> I think we should group as per the domain, eg auth reducer could be one, messages reducer, settings reducer etc.

Yeah that would make it much easier for me to reuse the existing code from the old flux store.
